be/src/olap/rowset/segment_v2/binary_plain_page.h

```diff
@@ -80,24 +80,29 @@ class BinaryPlainPageBuilder : public PageBuilder {
     Slice finish() override {
         _finished = true;
 
-        size_t offsets_pos = _buffer.size();
-
-        // Set up the header
-        encode_fixed32_le(&_buffer[0], _offsets.size());
-        encode_fixed32_le(&_buffer[4], offsets_pos);
-
+        // Set up trailer
+        size_t new_cap = _buffer.size() + (_offsets.size() + 1) * sizeof(uint32_t);
+        std::cout << "cap0:" << _buffer.capacity() << std::endl;
```

Comment:
Why using `std::cout` to print debug log?

Reply:
I'm delete the debug log after update the commit,Now I'm debuging with some problem